src/ILAMB/ConfCO2.py

```diff
@@ -11,6 +11,13 @@
 import pylab as plt
 import numpy as np
 import os
+import copy
+from ILAMB import ccgfilt
+import scipy
+import dateutil
```

Comment:
Unused import

Reply:
This didn't come out as clear as when I clicked the line on the review. The `csv` and `dateutil` imports are unused I think.